<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#000000  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_dot">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_division">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_addition">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multiplcation">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_subtraction">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rightParenthesis">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_leftParenthesis">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#F50A3F  ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_c">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_arrows">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
lineEdit.setStyleSheet(&quot;color:red&quot;);

font: 75 28pt &quot;Aharoni&quot;;

lineEdit.setStyleSheet(&quot;background-color:#000000&quot;);


lineEdit.setFont(QFont( &quot;Timers&quot; , 28 ,  QFont::Bold) );
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_show">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
lineEdit.setStyleSheet(&quot;color:red&quot;);

font: 75 28pt &quot;Aharoni&quot;;

lineEdit.setStyleSheet(&quot;background-color:red&quot;);


lineEdit.setFont(QFont( &quot;Timers&quot; , 28 ,  QFont::Bold) );
</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remainder">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_nagative">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background-color:#006699    ;
color: white;   
border-radius: 10px;  
border: 2px groove gray;
border-style: outset;}
QPushButton:hover{background-color:white; color: black;}
QPushButton:pressed{background-color:rgb(85, 170, 255);
border-style: inset; 
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>254</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menu_quit">
    <property name="title">
     <string>选择</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_quit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
